src/Options.js

```diff
@@ -18,20 +37,10 @@ function Options(data) {
     this.style = data.style;
```

Comment:
Good catch. The whole comment refers to “parent” functionality that is no longer present or needed. I'll replace this comment with a comment about it being readonly.